script/gen_requirements_all.py

```diff
@@ -26,6 +26,8 @@
     'homeassistant.components.recorder.models',
 )
 
+IGNORE_PIN = ('colorlog>2.1,<3', 'keyring>=9.3,<10.0', 'urllib3')
```

Comment:
- colorlog and keyring lives in `homeassistant.scripts`
- [urllib](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/device_tracker/unifi.py#L18) is a special case